src/styles/variables.ts

```diff
@@ -69,7 +69,9 @@ export default {
     fontSizeSignInHeroMedium: 38,
     fontSizeSignInHeroXSmall: 26,
     fontSizeSignInHeroSmall: 28,
+    fontSizeSignInHeroSmallLineHeight: 30.8,
```

Comment:
Where is this value coming from? We usually do base4 for all of our numerical values.

Reply:
Another unused value left from early implementations, I removed it.